opslevel.yml

```diff
@@ -0,0 +1,13 @@
+version: 1
+service:
+  name: Performance challenge
+  description: Hiring challenge for Performance engineering positions
+  owner: quality
+  tags:
+    - key: type
+      value: hiring-challenge
```

Comment:
@jsfr this will trigger a "maturity" error in OpsLevel because the value is not one of the predefined ones. Can you please advise on how we should proceed here? See also the related question in Notion doc: https://www.notion.so/How-to-configure-a-service-in-OpsLevel-f6483fcb4fdd4dcc9fc32b7dfe14c262?d=35dfc39596d847e498ba373da8ee0727#db46575cb9d54c7a956b033ac7fc6225

Reply:
We're already discussing about that here: https://github.com/pleo-io/heracles/pull/15/files#r950056636

Reply:
DevExp team will create a new `hiring-challenge` type, see: https://github.com/pleo-io/heracles/pull/15#discussion_r951302084